ntropy_sdk/ntropy_sdk.py

```diff
@@ -1884,34 +1609,27 @@ def _add_transactions_chunk(
         return transactions_enriched
 
     @staticmethod
-    def _build_params_str(model_name: str = None) -> str:
-        params = {}
-        if model_name is not None:
-            params["model_name"] = model_name
-
-        params_str = urlencode(params)
-        return params_str
+    def _build_params_str() -> str:
+        return urlencode({})
 
```

Comment:
this doesn't do anything useful anymore, we should just remove it

Reply:
done